docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -2243,24 +2246,24 @@ and depending on the progress made, indexed into different indices.
 [[set-processor]]
 === Set Processor
```

Comment:
<img width="771" alt="image" src="https://user-images.githubusercontent.com/976291/47043129-75e0f280-d152-11e8-8583-a07932a7167d.png">

Reply:
<img width="791" alt="image" src="https://user-images.githubusercontent.com/976291/47232023-2ee23f80-d394-11e8-84c7-946c705ebba9.png">

@tomcallahan @debadair ^^ example using description instead of superscript. Thoughts on which you prefer seeing side by side ?

Reply:
I like this the best of the available options